src/components/table/TableCellEdit.vue

```diff
@@ -1,11 +1,13 @@
 <template>
   <div class="table-cell-editable">
     <input type="checkbox" @change="(event) => selectItem(event)" />
+    <!-- Edge does not respect lang="en-US" as such to not have ',' as decimal separator we need type="text" -->
     <input
       :ref="`${props.item.date}-${props.id}-value`"
       v-model.number="currentItem.y"
       class="table-cell-edit"
-      type="number"
+      type="text"
+      inputmode="numeric"
```

Comment:
Should this not be `decimal` to show the decimal separator on the keyboard (on mobile)?

Reply:
Updated